templates/data/istio-module-data.yaml

```diff
@@ -118,15 +117,15 @@ templates:
                     - image_buildpack-golang # takes latest golang image
                     - jobConfig_default
                     - jobConfig_postsubmit
+                    - jobConfig_slack_goat
                   local:
                     - job_default
-                    - job_post
```

Comment:
we do not want slack notifications on post jobs?

Reply:
We want this, since otherwise failing jobs on main might go unnoticed. Also I didn't change the configuration, this is the same [that was in place before](https://github.com/kyma-project/test-infra/blob/28437186a420ec49df6bbcc36d514c5aba922ba5/prow/jobs/modules/internal/istio-manager.yaml#L197), but I only removed the `job_post` local config (that only applied the Slack config) with a global config.

Reply:
now I get it, there is global config for this - `jobConfig_slack_goat`